src/main/java/com/intel/genomicsdb/importer/GenomicsDBImporter.java

```diff
@@ -601,17 +601,21 @@ private static GenomicsDBImporter createImporter(final ParallelImportConfig para
                                                      final ChromosomeInterval chromInterval) throws IOException {
         String arrayName = String.format(CHROMOSOME_INTERVAL_FOLDER, chromInterval.getContig(),
                 chromInterval.getStart(), chromInterval.getEnd());
-        GenomicsDBImportConfiguration.GATK4Integration gatk4Integration = parallelImportConfig.getImportConfiguration()
-                .getGatk4IntegrationParameters().toBuilder().setLowerSampleIndex((long) index)
-                .setUpperSampleIndex((long) (index + parallelImportConfig.getBatchSize() - 1))
-                .setUseSamplesInOrderProvided(true).build();
+        //GenomicsDBImportConfiguration.GATK4Integration gatk4Integration = parallelImportConfig.getImportConfiguration()
+            //.getGatk4IntegrationParameters();
+                //.getGatk4IntegrationParameters().toBuilder().setLowerSampleIndex((long) index)
+                //.setUpperSampleIndex((long) (index + parallelImportConfig.getBatchSize() - 1))
+                //.setUseSamplesInOrderProvided(true).build();
+        parallelImportConfig.setImportConfiguration(parallelImportConfig.getImportConfiguration().toBuilder()
```

Comment:
You should assign values to lb and ub apter partition (line 615) due to ImportConfiguration builder is created again at that point.